src/ui/components/RecipeIngredient/index.ts

```diff
@@ -0,0 +1 @@
+export { default as RecipeIngredient } from './RecipeIngredient';
```

Comment:
add these imports in `components/index.ts` so you only import from `@ui/components`